agent/tests/Integration/PharTest.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+it('can start the agent and authenticate', function () {
```

Comment:
Does it make an HTTP request to the real API endpoint?

Reply:
The Integration test does.

I'll Slack you about it.